dev/NumberBox/NumberBox.h

```diff
@@ -98,7 +98,7 @@ class NumberBox :
     bool m_valueUpdating{ false };
     bool m_textUpdating{ false };
 
-    winrt::SignificantDigitsNumberRounder m_displayRounder{};
+    winrt::IncrementNumberRounder m_displayRounder{};
```

Comment:
I initially didn't consider how this would work with >1 integer digit. Switching to a `IncrementNumberRounder` ensures that it always formats it with 6 fractional digits no matter how many integer digits it has.